pkg/kots/rego/kots-spec-opa-nonrendered.rego

```diff
@@ -553,6 +566,119 @@ lint[output] {
   }
 }
 
+# Check if repeatable ConfigOption has a template field defined
+lint[output] {
+  config_option := config_options[_]
+  item := config_option.item
+  item.repeatable
+  not item.template
+  field := concat(".", [config_option.field, "type"])
+  message := sprintf("Repeatable Config option \"%s\" is missing the template field", [string(item.name)])
+  output := {
+    "rule": "repeat-config-option-missing-template",
+    "type": "error",
+    "message": message,
+    "path": config_file_path,
+    "field": field,
+    "docIndex": config_data.docIndex
+  }
+}
+
+# Check if repeatable ConfigOption has at least one template
+lint[output] {
+  config_option := config_options[_]
+  item := config_option.item
+  item.repeatable
+  item.template == null
+  field := concat(".", [config_option.field, "type"])
+  message := sprintf("Repeatable Config option \"%s\" is missing a template target", [string(item.name)])
+  output := {
+    "rule": "repeat-config-option-missing-template",
```

Comment:
this has the same rule name as the previous one, i recommend that each rule has a unique name

Reply:
too much copy/paste :)